### ISSUE TITLE
The iso is not being created

### ISSUE THREAD

**Issue by Dominik159E0** [ntdevlabs/tiny11builder#280]
Trying the utility, everything works, until the last step, the making of the iso, it just dies that it cannot find the boot sector file.
This happens on the latest release and the one before:
![obrázok](https://github.com/user-attachments/assets/78e2e869-802d-49eb-a1c0-57072c0649c1)

**Comment by 0n1cOn3** [comment]
Can you check if the file indeed is missing or if it is there. That could determine where to look in the script.

**Comment by Dominik159E0** [comment]
> Can you check if the file indeed is missing or if it is there. That could determine where to look in the script.

Sorry for the delay but, the file does exist
![obrázok](https://github.com/user-attachments/assets/8c5512c8-aa75-470a-8a44-0a161b3fba4c)
My guess is that it does not add C: onto the beggining of the path perhaps?

**Comment by AsharLohmar** [comment]
That command is the same in both scripts and for whatever reason it doesn't fail in the other one ... maybe it's one of the vars that it's not defined or non correctly defined ...

**Comment by AsharLohmar** [comment]
@Dominik159E0 you were right ... the command uses the variable `$ScratchDisk` but in the `tiny11Coremaker.ps1` script the variable is not defined anywhere, that's the only line where this variable is mentioned. For the other pourposes the script uses `$mainOSDrive` ...

**Comment by frosty3907** [comment]
same issue, fix?

**Comment by Dominik159E0** [comment]
tldr for others in the future:
Just open the script in notepad, then CTRL + H (aka Find and Replace)
Find:  `$ScratchDisk`
Replace: `$mainOSDrive`

Closing this issue with this message
